pkg/invoker/restoresession.go

```diff
@@ -312,28 +316,116 @@ func (inv *RestoreSessionInvoker) UpdateStatus(status RestoreInvokerStatus) erro
 	return nil
 }
 
+func (inv *RestoreSessionInvoker) GetSummary(target v1beta1.TargetRef, session kmapi.ObjectReference) *v1beta1.Summary {
+	summary := &v1beta1.Summary{
+		Name:      session.Name,
+		Namespace: session.Namespace,
+		Target:    target,
+		Invoker: core.TypedLocalObjectReference{
+			APIGroup: pointer.StringP(v1beta1.SchemeGroupVersion.Group),
+			Kind:     v1beta1.ResourceKindRestoreSession,
+			Name:     inv.restoreSession.Name,
+		},
+	}
+	restoreSession, err := inv.stashClient.StashV1beta1().RestoreSessions(session.Namespace).Get(context.TODO(), session.Name, metav1.GetOptions{})
+	if err != nil {
+		summary.Status.Phase = string(v1beta1.RestoreFailed)
```

Comment:
Can we return anything like `unknown` here?

Reply:
done